onnx/test/model_inference_test.py

```diff
@@ -26,6 +26,28 @@ def _check(self, model_text: str, *expected: int):
             elem_type = tensor_type.elem_type
             self.assertEqual(elem_type, expected_elem_type)
 
+    def _check_inference_error(self, model_text: str):
```

Comment:
Looks unused?

Reply:
Will leave it in, as it will be useful when we add tests for things that should cause an error (I undid such a test-case pending our discussion elsewhere about handling undefined ops).